app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminAssignCategoryToProductAndSaveActionGroup.xml

```diff
@@ -8,17 +8,9 @@
 
 <actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
               xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
-    <actionGroup name="AdminAssignCategoryToProductAndSaveActionGroup">
-        <arguments>
-            <argument name="categoryName" type="string"/>
-        </arguments>
-        <!-- on edit Product page catalog/product/edit/id/{{product_id}}/ -->
-        <click selector="{{AdminProductFormSection.categoriesDropdown}}" stepKey="openDropDown"/>
-        <checkOption selector="{{AdminProductFormSection.selectCategory(categoryName)}}" stepKey="selectCategory"/>
-        <click selector="{{AdminProductFormSection.done}}" stepKey="clickDone"/>
-        <waitForPageLoad stepKey="waitForApplyCategory"/>
+    <actionGroup name="AdminAssignCategoryToProductAndSaveActionGroup" extends="AdminAssignCategoryToProductActionGroup">
```

Comment:
That's a very good cleanup. But unfortunately, it makes the PR backward-incompatible for 2.4-develop branch. There might be 3rd-party modules that extend the current action group and rely on the step keys. By removing the existing step keys we may brake such systems